Makefile

```diff
@@ -38,7 +38,7 @@ env:
 
 # Ignoring generated ones with .py extension.
 lint:
-	pylint -rn qiskit test
+	pylint -j 2 -rn qiskit test
```

Comment:
Could use `-j 0` (pylint will auto-detect CPU count)?

Reply:
Also, looks like there are failures for missing monkey-patched gates (presumably due to changing the evaluation order, `pylint test/ qiskit/` also raises these.)

Reply:
Is `ignored-classes=QuantumCircuit` too much? I can clean up all the around ` # pylint: disable=no-member`

Reply:
`ignored-classes=QuantumCircuit` seems reasonable, from the doc, QuantumCircuit fits its purpose well:
```
    --ignored-classes=<members names>
                        List of class names for which member attributes should
                        not be checked (useful for classes with dynamically
                        set attributes). This supports the use of qualified
                        names. [current:
                        optparse.Values,thread._local,_thread._local]
```